Code/GraphMol/Descriptors/Data3Ddescriptors.h

```diff
@@ -0,0 +1,30 @@
+#include <string>
+
+using namespace std;
+
+#ifndef _Data3Ddescriptors
+
+#define _Data3Ddescriptors
+
+class Data3Ddescriptors
+{
+   private:
+		static double mw[53];
+		static double vdW[83];
+		static double neg[53];
+		static double pol[53];
+		static double ionpol[83];
+		static double rcov[83];
+
+   public:
+	   Data3Ddescriptors();
+	   double* getMW();
+	   double* getVDW();
+	   double* getNEG();
+	   double* getPOL();
+	   double* getIonPOL();
+	   double* getRCOV();
```

Comment:
I think some comments are required here. This is not MW, so  a different name or a comment would be useful. Also, where do the values come from.